build.gradle

```diff
@@ -233,3 +335,34 @@ task updateMasterLanguageFile(type: JavaExec, dependsOn: 'classes') {
 // jar.dependsOn(updateMasterLanguageFile)
 
 idea { module { inheritOutputDirs = true } }
+
+artifacts {
+    archives devJar, sourceJar, changelog.output
+}
+
+if (!hasProperty("mavenUsername")) {
+    ext.mavenUsername="${System.getenv().MAVEN_USERNAME}"
+}
+
+if (!hasProperty("mavenPassword")) {
+    ext.mavenPassword="${System.getenv().MAVEN_PASSWORD}"
+}
+
+if (!hasProperty("mavenURL")) {
+    ext.mavenURL="${System.getenv().MAVEN_URL}"
+}
+
+//if ( !project.buildInfo.tag.contains("-dirty") && (System.getenv().ONLY_RELEASES!="true" || project.buildInfo.isRelease) ) {
+//    task testIfUploadable {}
```

Comment:
Nit: Does this commented code need to exist?

Reply:
Doesn't need to exist, but left it in as an example to prevent accidental releases (see GT6's buildscript), can delete if you don't care about such checks.